peekaboo/server.py

```diff
@@ -101,6 +104,12 @@ def shutdown(self):
 
         socketserver.ThreadingUnixStreamServer.shutdown(self)
 
+        # FIXME: This is spaghetti! Signal cuckoo module to shut down so the
```

Comment:
@jack28: What do you think of this? Do we want the functionality so badly that we accept this spaghetti code or can we hold off until embed mode has been retired for good?

Reply:
Why don't we kill it now?

Reply:
I know of at least one user who still has to convert away from it and will likely discover some corner cases to improve in api mode still. We should IMO have a deprecation period (which is why I opened #126 today).

Reply:
Shutdown command postponed and removed from this PR.